CMakeLists.txt

```diff
@@ -7,6 +7,27 @@ set(SRC "${CMAKE_CURRENT_SOURCE_DIR}")
 set(CMAKE_POSITION_INDEPENDENT_CODE ON) # This does not set -fPIC :(
 set(CMAKE_CXX_FLAGS "-fPIC")
 
+## ZSerio binary
+####
+option(ZSERIO_JAR_PATH "Path to zserio jar.")
+if (ZSERIO_JAR_PATH)
+  set(ZSERIO "java")
```

Comment:
Consider using `find_package(Java REQUIRED COMPONENTS Runtime)` and then `Java_JAVA_EXECUTABLE`

Reply:
Using `find_program` should be enough, we need java at compile (configure) time only.